Documentation/PageTsconfig/Mod/Wizards.rst

```diff
@@ -59,19 +83,20 @@ newContentElement.wizardItems
             :type: string, comma-separated list of items
             :Path: mod.wizards.newContentElement.wizardItems.[group].show
 
-            Name of the group.
-
-            Comma-separated list of items to show in the group. Use `*` to show
-            all, example:
-
-            ..  code-block:: typoscript
-                :caption: EXT:site_package/Configuration/page.tsconfig
+            ..  versionchanged:: 13.0
+                The configuration of the New Content Element Wizard has been
+                changed to automatically registering the groups and elements
+                from the TCA configuration.
 
-                # Hide bulletList
-                mod.wizards.newContentElement.wizardItems.common.show := removeFromList(bullets)
-                # Only show text and textpic in common
-                mod.wizards.newContentElement.wizardItems.common.show = text,textpic
+                The previously used option to show / hide elements
+                :typoscript:`mod.wizards.newContentElement.wizardItems.<group>.show` is
+                not evaluated anymore.
 
+                All configured groups and elements are automatically shown. Removing these
+                groups and elements from the New Content Element Wizard can be done via
+                the option :confval:`removeItems <mod-wizards-newContentElement-wizardItems-removeItems>` and
+                :confval:`[group].removeItems <mod-wizards-newContentElement-wizardItems-group-removeItems>`
+                options.
```

Comment:
Duplicate? I'm unsure

Reply:
Nope, one is for the group, the other is in general, two different options